calculation_functors.h

```diff
@@ -1,38 +1,10 @@
 #pragma once
 
-#ifdef __CUDACC__
```

Comment:
Everything deleted here was dead code, only compiled in the SCHRODINGER installation, which is a mistake